pkg/kopia/command/server_test.go

```diff
@@ -41,6 +45,8 @@ func (kServer *KopiaServerTestSuite) TestServerCommands(c *C) {
 			f: func() []string {
 				args := ServerStartCommandArgs{
 					CommandArgs:          commandArgs,
+					CacheArgs:            cacheArgs,
+					CacheDirectory:       "cache/dir",
```

Comment:
we are passing values of these fields, same everywhere. We should change that in every test case with diff possible values.

Reply:
@viveksinghggits But on other hand, we're also keeping the `commandArgs` same for all tests, which contains details like `RepoPassword`, `ConfigFilePath` & `LogDirectory`. So thats's why I also kept the `cacheArgs` same for all tests. But if you want we can keep them different. WDYT ?

Reply:
I think it should be ok, even if we kept them the same.